administration/troubleshooting.md

```diff
@@ -0,0 +1,311 @@
+# Troubleshooting
+
+* [Tap Functionality: generate events or records](troubleshooting.md#tap-functionality)
+* [Dump Internals Signal](troubleshooting.md#dump-internals--signal)
+
+## Tap Functionality
+
+Tap can be used to generate events or records detailing what messages
+pass through Fluent Bit, at what time and what filters affect them.
+
+To test out Fluent Bit Tap we will be using a publicly accessible docker container: 'fluent/fluent-bit:latest
+'.
+
+### Simple example
+
+First we will make sure that the container we are going to use actually supports Fluent Bit Tap (available in Fluent Bit 2.0+):
+
+```shell
+$ docker run --rm -ti ghcr.io/calyptia/core/calyptia-fluent-bit:main --help | grep -Z
+  -Z, --enable-chunk-trace     enable chunk tracing. activating it requires using the HTTP Server API.
+```
+
+If the `--enable-chunk-trace` option is present it means Fluent Bit has support for Fluent Bit Tap but it is disabled by default, so remember to enable it with this option.
+
+Tap support is enabled and disabled via the embedded web server, so enable it like so (or the equivalent option in the configuration file):
+
+```shell
+$ docker run --rm -ti -p 2020:2020 ghcr.io/calyptia/core/calyptia-fluent-bit:main -Z -H -i dummy -p alias=input_dummy -o stdout -f 1
+Calyptia Fluent Bit v22.8, patch_level=1
```

Comment:
run the docker run command and replace this log with the new one (upstream version info)